modules/login/php/requestaccount.class.inc

```diff
@@ -195,6 +240,8 @@ class RequestAccount extends \NDB_Form
                 $err['captcha'] = 'Please complete the reCaptcha!';
             }
         }
+        // Show success message even if email already exists for security reasons
+        $this->tpl_data['success'] = "some text";
```

Comment:
?

Reply:
@driusan 
For some reason, having `true` as a boolean does not work, the `{if $success}` statement in the tpl is not entered.

havin `"true"` as text does work... but is confusing... I thought of putting `"This text is completely useless but is more true than the boolean true"` but that would be pushing it!